app/collections/playlists.js

```diff
@@ -1,9 +1,44 @@
 import Backbone from 'backbone';
 import Playlist from '../models/playlist';
+import cozysdk from 'cozysdk-client';
 
 
 const Playlists = Backbone.Collection.extend({
-    model: Playlist
+   	
+   	model: Playlist,
+
+    initialize() {
+        this.on('add', this.onAdd, this);
```

Comment:
I don't understand how this one is useful: if you want to ensure a Model is saved on the server when added to a collection, juste use `Collection.create` instead of `Collection.add`. Is there a use case here?